testing/src/main/java/org/oppia/android/testing/espresso/TextInputAction.kt

```diff
@@ -0,0 +1,66 @@
+package org.oppia.android.testing.espresso
+
+import android.view.View
+import androidx.test.espresso.matcher.BoundedMatcher
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+
+/**
+ * Action for matching error text and performing other actions on [TextInputLayout] in a test
+ * infrastructure-specific way.
+ */
+class TextInputAction {
+
+  companion object {
+    /**
+     * Checks that the TextInputLayout view has the [expectedErrorText] error text.
+     *
+     * @returns a [BoundedMatcher] that matches the [expectedErrorText] with the TextInputLayout's
+     *     error text
+     */
+    fun hasErrorText(expectedErrorText: String): BoundedMatcher<View, TextInputLayout> {
+      return ErrorTextExisted(expectedErrorText)
```

Comment:
This class should probably be ErrorTextExistsMatcher and the other ErrorTextDoesNotExistMatcher for correct verb tense & for consistency with how Espresso matchers are usually named.

Reply:
Done.